app/DynamicsAdapter/DynamicsAdapter.Web/SearchRequest/AutoCloseSearchRequestJob.cs

```diff
@@ -57,7 +57,15 @@ public async Task Execute(IJobExecutionContext context)
                         ssgSearchRequest.AutoCloseStatus == SearchRequestAutoCloseStatusCode.CPMissingData.Value)
                     {
                         _logger.LogInformation("Autoclose: Calling ssg_SearchRequestCreateCouldNotAutoCloseNote SearchRequestId=" + ssgSearchRequest.SearchRequestId + " AutoCloseStatus "+ssgSearchRequest.AutoCloseStatus+"");
-                        await _searchRequestService.SearchRequestCreateCouldNotAutoCloseNote(ssgSearchRequest.SearchRequestId);
+                        try
+                        {
+                            await _searchRequestService.SearchRequestCreateCouldNotAutoCloseNote(ssgSearchRequest.SearchRequestId, cts.Token);
+                        }
+                        catch (Exception e)
+                        {
+                            _logger.LogInformation("Autoclose: Encountered Unexpected Exception");
```

Comment:
Why 2 log messages?  Isn't a single log call enough?

```cs
_logger.LogError(e, "Autoclose: Encountered Unexpected Exception");
```

I also noticed a lot of the other log messages you added in the past few weeks are not properly using semantic logging. For example,

```cs
 _logger.LogInformation("Autoclose: Successfully Processed " + requestList.Count + " Records");
```

should record the count as a property on the log message.

```cs
 _logger.LogInformation("Autoclose: Successfully Processed {SearchRequestCount} Records", requestList.Count);
```